torchvision/prototype/utils/_internal.py

```diff
@@ -48,8 +48,17 @@
 class StrEnumMeta(enum.EnumMeta):
     auto = enum.auto
 
-    def __getitem__(self, item):
-        return super().__getitem__(item.upper() if isinstance(item, str) else item)
+    def from_str(self, member: str):
```

Comment:
Meta classes are fairly advanced Python. This is what you get when using `type(self)` of an instance of a class that is using this as meta class. This means, `self` is a subclass of `StrEnum` here. Thus, we don't use the `@classmethod` decorator here, since this is already a class method.

Reply:
I was about to comment on that. So we will be able to use it as class instance, right? See:
https://github.com/pytorch/vision/blob/93c85bbcc31f8d5a052daf06f2f91f39697af1a4/torchvision/prototype/models/_api.py#L50-L54

Reply:
Yes. Nothing changes for the subclass.